### PR TITLE
Digeq 1 sign in

### PR DESCRIPTION
**Pull request by weapdiv-david**
**What**

Basic login functionality and sign in screen

**How to test**

Automated tests available.  For manual testing visit the root URL and be redirected to login screen.  /logout url can be used to logout

**Who can test**

Anybody except @weapdiv-david or @LJBabbage

### CODE REVIEW

File: author/tests.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    def test_successful_login(self):
+        password = 'notANatural'
+        user = create_user(username='user', email='valid@gattaca.com', password=password)
+
+        login = self.client.login(username=user.username, password=password)
+
+        self.assertTrue(login)
+
+        response = self.client.get(reverse('welcome'), follow=True)
+        self.assertContains(response, 'welcome')
```
Comment (warren-methods) [inline author/tests.py]: newline at EOF?

**Comment by warren-methods** [comment]
I think it's worth squashing the commits, as there are a lot of non-descriptive small check ins. However the actual functionality looks good to my eyes. :8ball:

**Comment by dhilton** [comment]
Happy to review this when ready.

**Comment by warren-methods** [comment]
I still think we should squash these commits using an interactive rebase and then we should be able rebase to master. Hopefully that should allow us to have a one commit with single description then. At the minute the git history hurts my head.

**Pull request closed without merging**